### PR TITLE
Add hints to list-ops

### PR DESCRIPTION
**Pull request by archanid**
Maybe too wordy for now. We could remove the bullet points and just keep the first blurb.

see exercism/javascript/pull/677

### CODE REVIEW

**Comment by archanid** [comment]
Thanks for the review, but giving it more thought after actually doing the exercise, I feel like the single hint is sufficient. Talking about `.length` on the backing array and `array[indexer]` seems a bit too much. Thoughts?

**Comment by SleeplessByte** [comment]
I like it, but I think we want to _discourage_ backing arrays, so I should not mention it as such.
```markdown
Using core language features to build construct and desconstruct arrays via destructuring
as well as the array literal `[]` are allowed, but no functions from the `Array.prototype` 
should be used.
```

**Comment by archanid** [comment]
Makes sense @SleeplessByte how is it now?

**Comment by SleeplessByte** [comment]
Thank you <3

**Comment by SleeplessByte** [comment]
The failing test is fixed in https://github.com/exercism/typescript/pull/259 I believe, so I'll merge.
